home/views/data_sources.py

```diff
@@ -234,28 +235,34 @@ def _departments_data():
 
 class GenedData(View):
     def get(self, request):
-        geneds = request.GET["geneds"].replace("&", "").split("=on")
+        form_data = request.GET["geneds"].replace("&", "").split("=on")
+        geneds = form_data[:-1]
+        anyall = form_data[-1].split("=")[1]
```

Comment:
this is brittle as it relies on the order of the form data arguments. I've changed this to use 
 proper query string parsing from urllib.